tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py

```diff
@@ -117,14 +93,9 @@
 
     # add datsources now that variables are configured
     datasources = yaml.load(datasources_yaml)
-    my_postgres_db = context.add_datasource(
-        name="my_postgres_db", **datasources["datasources"]["my_postgres_db"]
+    context.sources.add_sql(
+        name="my_postgres_db", connection_string=os.get("my_postgres_db_yaml_creds")
```

Comment:
@Kilo59 wanting to check, is this a good way to get the `connection_string` from the environment to the `add_sql()` method?

Reply:
```python

context.sources.add_sql(
        name="my_postgres_db", connection_string=r"${my_postgres_db_yaml_creds}")
```